specs/eip4844/validator.md

```diff
@@ -128,26 +180,27 @@ Namely, the blob handling and the addition of `BlobsSidecar`.
 
 #### Constructing the `BeaconBlockBody`
 
-##### Blob commitments
+##### Blob KZG commitments
 
 After retrieving the execution payload from the execution engine as specified in Bellatrix,
 the blobs are retrieved and processed: 
 
-```python
+```
```

Comment:
Is this code snippet expected to be run by the validator?

I would make "are retrieved and processed" more as a direction rather than just a passive statement that it happens. "execute the following" or something